lefthook.yml

```
@@ -0,0 +1,66 @@
+pre-commit:
+  parallel: true
+  jobs:
+    - name: Format sidebars
+      glob: "files/sidebars/*.yaml"
+      run: yarn content fmt-sidebars
+      stage_fixed: true
+
+    - name: Fix redirects
+      glob: "files/en-us/_redirects.txt"
+      run: yarn content fix-redirects en-US && yarn content validate-redirects en-US
+      stage_fixed: true
+
+    - name: Format non-md
+      glob: "*.{js,json,yml,yaml,css,html}"
+      run: yarn prettier --ignore-unknown --write --cache {staged_files}
+      stage_fixed: true
+
+    - name: Sort cSpell dictionaries
+      glob: ".vscode/dictionaries/*.txt"
+      run: node scripts/sort_and_unique_file_lines.js {staged_files}
+      stage_fixed: true
+
+    - name: Run 'yarn filecheck'
+      glob: "*.{svg,png,jpeg,jpg,gif}"
+      run: yarn filecheck {staged_files}
+
+    - name: Check URL issues
+      glob: "*"
```

Why do we not restrict this to page changes?

```suggestion
      glob: "files/**/*.md"
```

---

Yeah good idea